octoprint_mrbeam/mrb_logger.py

```diff
@@ -84,11 +84,9 @@ def log(self, level, msg, *args, **kwargs):
         :type kwargs:
         """
 
-        try:
+        import io
+        if isinstance(msg, (str, io.IOBase)):
```

Comment:
As discussed offline:
- We could add some quick unit tests to make sure this is not crashing if msg is of type Exception for example and whatever other type you think it might be interesting.